resources/wikia/modules/tracker.js

```diff
@@ -161,13 +165,13 @@
 
 			// Set up params object - this should stay in sync with /extensions/wikia/Track/Track.php
 			params = {
-				'c': window.wgCityId,
-				'x': window.wgDBname,
-				'a': window.wgArticleId,
-				'lc': window.wgContentLanguage,
-				'n': window.wgNamespaceNumber,
+				'c': window.wgCityId || (wikiVariables && wikiVariables.id),
+				'x': window.wgDBname || (wikiVariables && wikiVariables.dbName),
+				'a': window.wgArticleId || (trackingDimensions && trackingDimensions[21]),
+				'lc': window.wgContentLanguage || (wikiVariables && wikiVariables.language.content),
+				'n': window.wgNamespaceNumber || (trackingDimensions && trackingDimensions[25]),
 				'u': userId,
-				's': window.skin,
+				's': window.skin || (data && data.kv_skin) || (trackingDimensions && trackingDimensions[4]),
```

Comment:
Is `(data && data.kv_skin)` needed?

Reply:
Not really, I am just greedy and I wanted to have it ALL 😶 Removed.